certbot-nginx/certbot_nginx/configurator.py

```diff
@@ -152,17 +152,17 @@ def deploy_cert(self, domain, cert_path, key_path,
                 "install a cert.")
 
         vhost = self.choose_vhost(domain)
-        cert_directives = [['ssl_certificate', fullchain_path],
-                           ['ssl_certificate_key', key_path]]
+        cert_directives = [['\n', 'ssl_certificate', ' ', fullchain_path],
```

Comment:
It's not really clear to me whether it was better to allow this change the semantics of `parser._add_directive` that required spaces to be explicit like this, or whether I should have implicitly prepended a `\n` and joined with `" "` every time it's called.  The current approach is perhaps less magical, but it has the downside of exposing both structures for the parse tree up here in `configurator.py`.